.travis.yml

```diff
@@ -20,6 +20,7 @@ before_install:
 
 before_script:
   - echo $ANDROID_HOME
+  - (while sleep 3; do echo "y"; done) | $ANDROID_HOME/tools/bin/sdkmanager "build-tools;26.0.2" "platforms;android-26"
```

Comment:
Can you separate this into another PR?

Reply:
Sure, but a bit later.

Reply:
Sure, take your time :) I'll look at this later after work :)

Reply:
@k0shk0sh done. #1134